convert_tapping_dataset_to_nirs.py

```diff
@@ -1,55 +1,139 @@
+import os
 import sys
-import re
+from collections import OrderedDict
 from pathlib import Path
 
+import click
 from pynwb import NWBHDF5IO
 
-# todo compile conversion tools into an installable package
-sys.path.append("src")
-from snirf import load_snirf, load_stim_table
+# Ensure modules in `src` are findable before importing
+sys.path.append(os.path.join(os.path.dirname(__file__), "src"))
+from bids import (
+    get_bids_version,
+    get_coordinate_frame_description,
+    get_dataset_authors,
+    get_sidecar_metadata,
+    get_subject_dir,
+    list_subject_ids,
+    load_stim_table,
+)
 from nwb import convert_to_nwb
+from snirf import get_format_version, load_snirf
+
+CONVERTER_VERSION = "0.1.0-dev.001"
+
+# copied from: https://github.com/rob-luke/BIDS-NIRS-Tapping/blob/master/README.md
+EXPERIMENT_DESCRIPTION = """This experiment examines how the motor cortex is activated
+during a finger tapping task. Participants are asked to either tap their left thumb to
+fingers, tap their right thumb to fingers, or nothing (control). Tapping lasts for 5
```

Comment:
```suggestion
fingers, tap their right thumb to fingers, or no cue is given (control). Tapping lasts for 5
```

Reply:
Probably not critical to say here - but even a cue to do nothing is an auditory stimulus and is a different kind of control condition.

Reply:
:+1: